packages/Search/examples/distributed_tree_driver/distributed_tree_driver.cpp

```diff
@@ -367,6 +366,14 @@ int main( int argc, char *argv[] )
     try
     {
         std::string node;
+        // NOTE Lame trick to get a valid default value
+#if defined( KOKKOS_ENABLE_SERIAL )
+        node = "serial";
+#elif defined( KOKKOS_ENABLE_OPENMP )
+        node = "openmp";
+#elif defined( KOKKOS_ENABLE_CUDA )
+        node = "cuda";
+#endif
```

Comment:
FWIW, I would switch the order to CUDA, OpenMP, Serial, so that you don't have to specify the node unnecessarily. If you compile with CUDA, you would run CUDA by default then.